OpenRA.Mods.Common/Activities/Air/Land.cs

```diff
@@ -21,18 +21,33 @@ public class Land : Activity
 		readonly Aircraft aircraft;
 		readonly bool requireSpace;
 		readonly Actor ignoreActor;
+		readonly WDist landAltitude;
+		readonly bool ignoreTarget;
 
 		bool landingInitiated;
 		bool soundPlayed;
 
-		public Land(Actor self, Target t, bool requireSpace, Actor ignoreActor = null)
+		public Land(Actor self, Target t, bool requireSpace, WDist landAltitude, Actor ignoreActor = null)
 		{
 			target = t;
 			aircraft = self.Trait<Aircraft>();
 			this.requireSpace = requireSpace;
```

Comment:
There are only three calls to `Land` left where `requireSpace = false` and those cases could (maybe should) be handled by using `ignoreActor` instead. So maybe it is a good idea to remove this parameter?

Reply:
Done.